src/components/User.vue

```diff
@@ -0,0 +1,242 @@
+						required>
+					<label for="password" class="infield">{{ t('registration', 'Password') }}</label>
+					<img id="password-icon"
+						class="svg input__icon"
+						:src="passwordIconPath"
+						alt="">
+					<Button class="toggle-password"
+						type="tertiary-no-background"
+						:aria-label="isPasswordHidden ? t('registration', 'Show password') : t('registration', 'Hide password')"
+						@click.stop.prevent="togglePassword">
+						<template #icon>
+							<Eye v-if="isPasswordHidden" :size="20" />
+							<EyeOff v-else :size="20" />
+						</template>
+					</Button>
+				</p>
+				<Button id="submit"
+					native-type="submit"
+					type="primary"
+					:wide="true"
+					:disabled="submitting"
+					@click="submit">
+					{{ t('registration', 'Create account') }}
+				</Button>
+			</fieldset>
+		</form>
+	</div>
+</template>
+
+<script>
+import { getRequestToken } from '@nextcloud/auth'
+import Button from '@nextcloud/vue/dist/Components/Button'
+import { generateFilePath } from '@nextcloud/router'
+import { loadState } from '@nextcloud/initial-state'
+import Eye from 'vue-material-design-icons/Eye'
+import EyeOff from 'vue-material-design-icons/EyeOff'
+
+export default {
+	name: 'User',
+
+	components: {
+		Button,
+		Eye,
+		EyeOff,
+	},
+
+	data() {
+		return {
+			email: loadState('registration', 'email'),
+			emailIsLogin: loadState('registration', 'emailIsLogin'),
+			emailIsOptional: loadState('registration', 'emailIsOptional'),
+			loginname: loadState('registration', 'loginname'),
+			fullname: loadState('registration', 'fullname'),
+			showFullname: loadState('registration', 'showFullname'),
+			enforceFullname: loadState('registration', 'enforceFullname'),
+			phone: loadState('registration', 'phone'),
+			showPhone: loadState('registration', 'showPhone'),
+			enforcePhone: loadState('registration', 'enforcePhone'),
+			message: loadState('registration', 'message'),
+			password: loadState('registration', 'password'),
+			additionalHint: loadState('registration', 'additionalHint'),
+			requesttoken: getRequestToken(),
+			loginFormLink: loadState('registration', 'loginFormLink'),
+			isPasswordHidden: true,
+			passwordInputType: 'password',
+			submitting: false,
+		}
+	},
+
+	computed: {
+		emailIconPath() {
+			return generateFilePath('core', 'img', 'actions/mail.svg')
+		},
+		phoneIconPath() {
+			return generateFilePath('core', 'img', 'clients/phone.svg')
+		},
+		userIconPath() {
+			return generateFilePath('core', 'img', 'actions/user.svg')
+		},
+		authIconPath() {
+			return generateFilePath('core', 'img', 'categories/auth.svg')
+		},
+		passwordIconPath() {
+			return generateFilePath('core', 'img', 'actions/password.svg')
+		},
+	},
+	methods: {
+		togglePassword() {
+			if (this.passwordInputType === 'password') {
+				this.passwordInputType = 'text'
+			} else {
+				this.passwordInputType = 'password'
+			}
+		},
+		submit() {
+			this.submitting = true
+		},
```

Comment:
????

Reply:
it's a form, I'm just trying to prevent sending the form twice